src/landing-page/components/LandingPage/index.tsx

```diff
@@ -1,33 +1,32 @@
 import { faUpRightFromSquare } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { Button, Tooltip } from "antd";
-import React, { ReactElement } from "react";
+import React, { ReactElement, useEffect, useState } from "react";
 
 import { landingPageContent } from "./content";
 import { DatasetEntry, ProjectEntry, ViewerArgs } from "../../types";
 import styled from "styled-components";
 import { FlexColumnAlignCenter, FlexColumn, FlexRowAlignCenter, VisuallyHidden, FlexRow } from "./utils";
 
+const MAX_CONTENT_WIDTH_PX = 1060;
+
 const Banner = styled(FlexColumnAlignCenter)`
   position: relative;
   --container-padding-x: 20px;
-  padding: 30px var(--container-padding-x);
+  padding: 40px var(--container-padding-x);
   overflow: hidden;
   margin: 0;
 `;
 
 const BannerTextContainer = styled(FlexColumn)`
   --padding-x: 30px;
-  padding: var(--padding-x);
-  max-width: calc(1060px - 2 * var(--padding-x));
+  padding: 26px var(--padding-x);
+  max-width: calc(${MAX_CONTENT_WIDTH_PX}px - 2 * var(--padding-x));
 
   --total-padding-x: calc(2 * var(--padding-x) + 2 * var(--container-padding-x));
   width: calc(90vw - var(--total-padding-x));
   border-radius: 5px;
-  // Fallback in case color-mix is unsupported.
-  background-color: var(--color-background);
-  // Make the background slightly transparent. Note that this may fail on internet explorer.
-  background-color: color-mix(in srgb, var(--color-background) 80%, transparent);
+  background-color: var(--color-landingpage-banner-highlight-bg);
```

Comment:
Nice and clean change of setting a direct background color! Just checking, we're good with dropping the `color-mix` fallback, right?

Reply:
Yup! The `--color-landingpage-banner-highlight-bg` already has transparency built-in.